lib/wasi/src/lib.rs

```diff
@@ -76,6 +80,31 @@ impl WasiEnv {
         ))
     }
 
+    /// Like `import_object` but containing all the WASI versions detected in
+    /// the module.
```

Comment:
I would rename the function, something like `import_object_for_all_wasi_versions`, it's better for discoverability, maybe?

Reply:
Yeah, the one reason I didn't use the term `import object` is that the first function returns a type called `ImportObject` and this one doesn't

Reply:
I'll just go with your suggestion for now 🤷